Competition2017/src/competition/subsystems/drive/DriveSubsystem.java

```diff
@@ -148,7 +163,6 @@ public void tankDrivePowerMode(double leftPower, double rightPower) {
      * @param motor used to access the motor to set and give data
      */
     private void updateMotorConfig(XCANTalon motor) {
-        motor.configEncoderCodesPerRev((int)encoderCodesProperty.get());
```

Comment:
My most recent read through of the docs suggests that it is not.

Motion Control Profile (MCP) value are saved locally, and those values are:
P Gain: Kp constant to use when control mode is a closed-loop mode.
I Gain: KI constant to use when control mode is a closed-loop mode.
D Gain: KD constant to use when control mode is a closed-loop mode.
F Gain: KF constant to use when control mode is a closed-loop mode.
I Zone: Integral Zone. When nonzero, Integral Accumulator is automatically cleared
when the absolute value of Closed-Loop Error exceeds it.
(Closed-Loop) Ramp Rate: Ramp rate to apply when control mode is a closed-loop
mode.
Allowable Closed-Loop Error: When Closed-Loop Error’s magnitude is less than this
signal, Integral Accum and motor output are auto-zeroed during closed-loop.
Peak Closed-Loop Output: Caps the maximal or peak motor-output during closed-loop.
Nominal Closed-Loop

this does not appear to include encoder configuration.